code/modules/research/xenoarchaeology/artifact/effect.dm

```diff
@@ -108,8 +108,13 @@ proc/GetAnomalySusceptibility(var/mob/living/carbon/human/H)
 	//latex gloves and science goggles also give a bit of bonus protection
 	if(istype(H.gloves,/obj/item/clothing/gloves/latex))
 		protected += 0.1
-
-	if(istype(H.glasses,/obj/item/clothing/glasses/science))
-		protected += 0.1
+	/*
+	If you have Anomaly Suit, Anomaly Hood, Latex Gloves AND Science Goggles, you will have
+		0.6 + 0.3 + 0.1 + 0.1 = 1.1
+	maximum protection level. And since I'm not sure that caller accepts well negative values
+	in return of this procedure ( 1 - 1.1 = -0.1 ) I'm going to do this check:
+	*/
+	if(protected < 1 && istype(H.glasses,/obj/item/clothing/glasses/science))
```

Comment:
As Arrow said, remove the first part of this if check (leaving only `if (istype(H.glasses, /obj/item/clothing/glasses/science))`, and add this right before return: `protected = CLAMP01(protected)`